rom/dev/src/rom_env.rs

```diff
@@ -70,10 +76,20 @@ pub struct RomEnv {
 
     /// PCR Bank
     pcr_bank: EnvCell<PcrBank>,
+
+    /// ICCM Range
+    iccm_range: Range<u32>,
 }
 
 impl Default for RomEnv {
     fn default() -> Self {
+        let iccm_start: u32;
+        let iccm_size: u32;
+        unsafe {
+            iccm_start = (&ICCM_ORG as *const u32) as u32;
+            iccm_size = (&ICCM_SIZE as *const u32) as u32;
```

Comment:
To be sound, you need to use [addr_of!](https://doc.rust-lang.org/stable/std/ptr/macro.addr_of.html) instead of using an intermediate reference. Probably best to do this everywhere you use this pattern.

Reply:
To be honest, creating extern "C" variables for stuff that is not actually memory is probably asking for trouble, since folks can cause unsound things to happen without writing unsafe. I'd prefer to replace ICCM_SIZE with ICCM_END, or make ICCM_SIZE a normal const defined 100% in Rust code.

Reply:
Added two constants ICCM_START and ICCM_SIZE.

Reply:
> To be sound, you need to use [addr_of!](https://doc.rust-lang.org/stable/std/ptr/macro.addr_of.html) instead of using an intermediate reference. Probably best to do this everywhere you use this pattern.
> 
> LLVM's optimizer is allowed to issue phantom loads from anything a reference points to (since normally that is safe). But if the reference is to something other than "normal memory", that is very dangerous.

Makes sense; will do this as a separate PR.